manifests/install.pp

```diff
@@ -1,11 +1,36 @@
 class ss_graphite_client::install inherits ss_graphite_client {
 
   package {
-    'realpath': ensure => installed;
     'logtail': ensure => installed;
+  }
+  
+  #package names are unstable between releases
+  if $facts['lsbdistcodename'] == 'jessie' {
+    package {
+    'realpath': ensure => installed;
     'logster': ensure => installed;
+    }
+  }
+  #logster no longer included as of buster
+  #so we can install via the stretch deb (last possible debian package version)
+  if $facts['lsbdistcodename'] == 'buster' {
+    file { '/opt/logster/logster_0.0.1-2_all.deb':
```

Comment:
`/opt/logster` doesn't exist - and opt is generally used to install optional software.

Can you please transfer and install the logster `.deb` file to and from `/usr/src` as a more suitable location.
eg. `file { '/usr/src/logster_0.0.1-2_all.deb':`

Reply:
Yup that's now amended to be `/usr/src`